cli/args/flags.rs

```diff
@@ -2725,7 +2736,13 @@ Specific version requirements to update to can be specified:
           .long("recursive")
           .short('r')
           .action(ArgAction::SetTrue)
-          .help("include all workspace members"),
+          .help("Include all workspace members"),
+      )
+      .arg(
+        Arg::new("json")
+          .long("json")
+          .action(ArgAction::SetTrue)
+          .help("Output outdated packages in JSON format")
```

Comment:
For the other `--json` flags, we've wanted to require people to specify the version of json output that they want similar to what Rust requires. This allows us to break the output over time while maintaining backwards compatibility.

If adding this flag, we should probably proactively do that here, but I'm not sure what that would look like. There's an open issue for this somewhere I think, but it's hard to search for.

Reply:
Do you mean that it would be specified on the cmd line, something like `--json v1`?

I just checked what `npm outdated --json` does and here is an example of the output it uses:

```
{
  "@types/node": {
    "current": "20.11.30",
    "wanted": "20.17.9",
    "latest": "22.10.1",
    "dependent": "deno-astro-adapter",
    "location": "/Users/irbull/git/deno/deno-astro-adapter/node_modules/@types/node"
  },
...
```
Do you think we should align with that format? It's a bit different, particularly that the package is the key.